tgui/packages/tgui/interfaces/AlertModal.js

```diff
@@ -18,11 +18,12 @@ import {
 } from 'common/keycodes';
 
 export class AlertModal extends Component {
-  constructor() {
+  constructor(autofocus = false) {
```

Comment:
This does not work the way you think it does, you should remove this from here, and perhaps just make it a constant.